control.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"log"
+)
+
+type StateController struct {
+	State  *AppState
+	Client *StatusGoClient
+}
+
+// For fetching current state of peers from status-go
+func (s *StateController) Fetch() {
+	peers, err := s.Client.getPeers()
+	if err != nil {
+		log.Panicln(err)
+	}
+	ps := s.State.GetData()
+	s.State.UpdatePeers(peers)
+	if ps.Current == -1 {
+		s.State.SetCurrentPeer(0)
+	}
+}
+
+// For removing a selected peer from connected to status-go
+func (s *StateController) RemovePeer(peer *Peer) error {
+	success, err := s.Client.removePeer(peer.Enode)
+	if err != nil || success != true {
+		log.Panicln(err)
```

Comment:
It will be `nil` if `success = false & err = nil`

Reply:
Good point.